src/components/Helper.re

```diff
@@ -0,0 +1,12 @@
+[@react.component]
```

Comment:
where we will use this component?

Reply:
this component is meant to display the status of something, like "Loading...", "No entries found", "Token expired"... we are already using the HelperTd on the Table component.

Reply:
top, got it!